### ISSUE TITLE
Moderate Vulnerability - gatsby-plugin-sharp

### ISSUE THREAD

**Issue by dg1234uk** [gatsbyjs/gatsby#12741]
**version**: Globally installed gatsby-cli 2.4.17 (gatsby-starter-default)
**version**: Package installed gatsby 2.2.6
**npm**: 6.9.0
**node**: v11.12.0

`npm audit` currently gives the following:

```
=== npm audit security report ===

┌──────────────────────────────────────────────────────────────────────────────┐
│                                Manual Review                                 │
│            Some vulnerabilities require your attention to resolve            │
│                                                                              │
│         Visit https://go.npm.me/audit-guide for additional guidance          │
└──────────────────────────────────────────────────────────────────────────────┘
┌───────────────┬──────────────────────────────────────────────────────────────┐
│ Moderate      │ Denial of Service                                            │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Package       │ js-yaml                                                      │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Patched in    │ >=3.13.0                                                     │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Dependency of │ gatsby-plugin-sharp                                          │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ Path          │ gatsby-plugin-sharp > svgo > js-yaml                         │
├───────────────┼──────────────────────────────────────────────────────────────┤
│ More info     │ https://npmjs.com/advisories/788                             │
└───────────────┴──────────────────────────────────────────────────────────────┘
```

`npm ls js-yaml` output

```
gatsby-plugin-sharp
└─┬ svgo@0.7.2
  └── js-yaml@3.7.0
```

**Comment by pieh** [comment]
Thanks for reporting this.

We are currently block on this, because `svgo` pins `js-yaml` version - and newest version of `svgo` uses `ja-yaml@3.12.0` (so still affected by it). This is issue on `svgo` repository to track https://github.com/svg/svgo/issues/1088

**Comment by dg1234uk** [comment]
Looks like this has been fixed as of gatsby-plugin-sharp@2.0.32. Its dependency versions are:

```
├─┬ gatsby-plugin-sharp@2.0.32
│ └─┬ svgo@1.2.0
│   └── js-yaml@3.13.0
```

Therefore I believe this issue can be closed.
